content/news/2021-08-19-gsoc-tools.md

```diff
@@ -0,0 +1,61 @@
++++
+title = "Devin's Google Summer of Code Ending"
+date = 2021-08-19
+extra.author = "Devin Ragotzy"
++++
+
+# A Summer of Ruma Tooling Comes to an End
+
+What a fun summer! I was hoping, this year I would become more a part of the Rust community, and
+I think I have. I learned how to open an MCP (Major Change Proposal), which is how Rust makes
+smaller changes to the language. I am involved in an effort to create a new plugin lint system
+similar to [Clippy]. I made a few more contributions to Clippy, fixing old lints and adding a new one.
+I am happy to become more involved in Ruma and Rust and plan to finish all my works in progress.
+
+Since I am working with more people than just my mentor, making progress seems to take longer.
+I am glad for the experience working with multiple people, learning how to follow directions from
+multiple people. You have to be able to write code in the "voice" of someone else or at least
+meet them halfway. I know that this is an invaluable skill that will serve me well in my career.
+Thank you to [@jplatte] and [@iinuwa] for all the help and encouragement!
+
+## Starting on Automation
+
+- [cargo-sort][DevinR528/cargo-sort] Some of the work on this happened before GSoC started but not all
+(I'm proud of this crate)
```

Comment:
I think the final blog post should clearly show what work you did over the coding period, so I think you'll have to be more specific than "some of the work happened before, but not all". Maybe add a sub-list with the changes you did for GSoC. If it wasn't PRs, try a commit range like this:

https://github.com/DevinR528/cargo-sort/compare/main@%7B2021-06-01%7D...main (date for this one was arbitrary)

Reply:
Fair point, they are kinda particular about the time periods.